docs/_pages/library.md

```diff
@@ -647,7 +647,7 @@ All functions in the `bit32` library treat input numbers as 32-bit unsigned inte
 function bit32.arshift(n: number, i: number): number
 ```
 
-Shifts `n` by `i` bits to the right (if `i` is negative, a left shift is performed instead). The most significant bit of `n` is propagated during the shift.
+Shifts `n` by `i` bits to the right (if `i` is negative, a left shift is performed instead). The most significant bit of `n` is propagated during the shift. When `i` is outside of `[-31..31]` range, returns an integer with all bits set to the sign bit of `n`.
```

Comment:
Is this actually true for values less than -31? For example, `bit32.arshift(-1,-100)` returns 0.

Reply:
Ah yes you're right of course. That text is only applicable to when an arithmetic right shift is performed.